packages/kbn-optimizer/limits.yml

```diff
@@ -94,7 +94,7 @@ pageLoadAssetSize:
   monitoring: 80000
   navigation: 37269
   newsfeed: 42228
-  observability: 100000
+  observability: 104448
```

Comment:
FYI @mistic The bundle increased significantly in https://github.com/elastic/kibana/pull/158678.  @dgieselaar is going to look if there is a way to decrease it, so this may be temporary.